### PR TITLE
Additional gRPC functionality + minor fixes for estimator

### PR DESCRIPTION
**Pull request by thowell**
Adds functionality to gRPC + Python to change the estimator's cost norms. 
Minor fixes.

### CODE REVIEW

File: grpc/estimator_service.cc [hunk 1]
```diff
@@ -339,26 +358,110 @@ grpc::Status EstimatorService::Weights(grpc::ServerContext* context,
...
-  for (int i = 0; i < ns; i++) {
+  for (int i = 0; i < num_sensor; i++) {
     output->add_sensor(estimator_.scale_sensor_[i]);
   }
 
   // force
-  int nv = estimator_.model_->nv;
-  if (input.force_size() == nv) {
-    mju_copy(estimator_.scale_force_.data(), input.force().data(), nv);
+  int num_jnt = 4;
```
Comment (erez-tom) [inline grpc/estimator_service.cc:371]: This hardcoded const seems wrong.

File: grpc/estimator_service.cc [hunk 2]
```diff
@@ -339,26 +358,110 @@ grpc::Status EstimatorService::Weights(grpc::ServerContext* context,
...
+    return {grpc::StatusCode::FAILED_PRECONDITION, "Init not called."};
+  }
+
+  // norm
+  estimator::Norm input = request->norm();
+  estimator::Norm* output = response->mutable_norm();
+
+  // set sensor type
+  int num_sensor = estimator_.num_sensor_;
+  if (input.sensor_type_size() == num_sensor) {
```
Comment (erez-tom) [inline grpc/estimator_service.cc:395]: if the python user made a mistake and got the number of sensors wrong, they may not understand why their call, intended as "set", isn't having the right effect.  Maybe we should bite the bullet and stick to the standard of having two proto calls, one for set and one for get?

File: python/mujoco_mpc/estimator_test.py [hunk 3]
```diff
@@ -246,7 +243,7 @@ def test_weights(self):
     self.assertTrue(np.linalg.norm(in_sensor_weight - weight["sensor"]) < 1.0e-5)
 
     ## force
-    in_force_weight = np.random.rand(model.nv)
+    in_force_weight = np.random.rand(4)
```
Comment (erez-tom) [inline python/mujoco_mpc/estimator_test.py:246]: same aberrant constant?

Comment (thowell) [reply python/mujoco_mpc/estimator_test.py:246]: See comment above.

**Review by erez-tom — APPROVED** [review]

**Pull request merged**